### PR TITLE
update dependencies for example

### PR DESCRIPTION
**Pull request by GJJ2019**
dependencies updating for example folder

- dio
- flutter_portal
- flutter_svg
- uuid

commenting un-use **mockito** dep in count example

### CODE REVIEW

**Comment by GJJ2019** [comment]
here example screenshot 
Screenshot (27)
Screenshot (28)
Screenshot (26)
Screenshot (25)

File: examples/marvel/lib/src/fake_marvel.dart [hunk 1]
```diff
@@ -27,30 +27,30 @@ class FakeDio implements Dio {
   @override
   Future<Response<T>> get<T>(
     String path, {
-    required Map<String, Object?> queryParameters,
+    Map<String, dynamic>? queryParameters,
```
Comment (rrousselGit) [inline examples/marvel/lib/src/fake_marvel.dart:30]: Why change this?

File: examples/marvel/lib/src/marvel.dart [hunk 2]
```diff
@@ -91,7 +91,7 @@ class MarvelRepository {
         )
         .toString();
 
-    final result = await _read(dioProvider).get<Map<String, Object>>(
+    final result = await _read(dioProvider).get<Map<String, dynamic>>(
```
Comment (rrousselGit) [inline examples/marvel/lib/src/marvel.dart:94]: Why change this?

File: examples/todos/pubspec.yaml [hunk 3]
```diff
@@ -13,13 +13,13 @@ dependencies:
...
   hooks_riverpod:
     path: ../../packages/hooks_riverpod
-  uuid: ^3.0.0-nullsafety.0
+  uuid: ^3.0.4
 
 dev_dependencies:
   build_runner: ^1.9.0
   flutter_test:
     sdk: flutter
-  mockito: ^5.0.0-nullsafety.5
```
Comment (rrousselGit) [inline examples/todos/pubspec.yaml:22]: Why is mockito commented?

Comment (GJJ2019) [reply examples/todos/pubspec.yaml:22]: its not getting use in count example if u want u can re-add it

File: examples/marvel/lib/src/fake_marvel.dart (hunk 1 above)
Comment (GJJ2019) [reply examples/marvel/lib/src/fake_marvel.dart:30]: its just change from new **dio** version 4.0.0 
https://pub.dev/documentation/dio/latest/dio/Dio/get.html

File: examples/marvel/lib/src/marvel.dart (hunk 2 above)
Comment (GJJ2019) [reply examples/marvel/lib/src/marvel.dart:94]: same as above

**Comment by GJJ2019** [comment]
are u planning something about this change
https://pub.dev/documentation/dio/latest/dio/Dio/get.html

**Pull request merged**